### ISSUE TITLE
Output Shape mismatch with original Yolov5 models

### ISSUE THREAD

**Issue by ZhanchongDeng** [ultralytics/yolov5#11977]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

I am trying to use ultralytics to download pretrained model/export them into different formats. However, I notice that using torch.hub.load and ultralytics.YOLO yields different model. I inspect the "u" models from ultralytics update, but found that yolov5\*u.pt produce **[1, 8400, 84]** whereas the original yolov5\*.pt produce **[1, 25200, 85]**. Even if I assume the new "u" model  make less max_prediction with the second channel, why is the last channel different? Which object class do the new model not support?

Below is a short code snippet to recreate the two models I am referring to. Also includes model summary from torchinfo to show their output size.
`pip install ultralytics torch torchinfo`

```
import torch
from torchinfo import summary
from ultralytics import YOLO

def main():
        # load original model
        model_torch = torch.hub.load('ultralytics/yolov5', 'yolov5s')
        # load ultralytics' model
        model_ultra = YOLO('yolov5s.pt')
        # now current directory has both yolov5s.pt and yolov5su.pt
        input_size = (1, 3, 640, 640)
        summary(model_torch.model, input_size)
        summary(model_ultra.model, input_size)

if __name__ == "__main__":
        main()
```

**Comment by ZhanchongDeng** [comment]
I guess it's YOLOv8 output. Should really make note of that.

**Comment by glenn-jocher** [comment]
@ZhanchongDeng thanks for bringing this up. It seems that the `u` models have been updated to yield fewer predictions and the output shape has indeed changed as you observed. The `u` models produce [1, 8400, 84] while the original models produce [1, 25200, 85]. 

The change was made to reduce the number of predictions and subsequently improve inference speed. The class mapping for the `u` models remains the same as for the original models, with the last channel being reserved for the object confidence score. 

The `u` models retain support for all object classes. Please refer to the Ultralytics Docs at https://docs.ultralytics.com/yolov5 for more detailed information on the YOLOv5 model outputs and the class labels.
